app/controllers/comments_controller.rb

```diff
@@ -159,4 +161,12 @@ def allow_modify_comments
     end
   end
 
+  def limit_comment
+     if current_user.newish? ||( current_user.newish? && current_user.assistant_curator?)
+       comments =  Comment.where(user_id: current_user.id , created_at: DateTime.now.to_date.beginning_of_day..DateTime.now.to_date.end_of_day)
```

Comment:
It is better to create scope for this case and similar ones (collections and etc..) which handles user's activity in a given day